### PR TITLE
Removing cms and profiles

### PR DESCRIPTION
**Pull request by pgwillia**
CMS moved to https://github.com/ualbertalib/library-cms  so profiles and cms can be removed from the discovery application.

#1280

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -40,10 +38,6 @@
   resources :bpsc_read_on_site_requests, only: [:new, :create]
   resources :rcrf_read_on_site_requests, only: [:new, :create]
 
-  root to: 'comfy/cms/content#show'
-
-  comfy_route :cms_admin, path: '/admin'
+  root to: 'catalog#index'
```
Comment (murny) [inline config/routes.rb]: Yay! We have a homepage again 🎊!

**Comment by pgwillia** [comment]
@nnunn I'm removing the cms and profile pages from discovery and reclaiming the root page. I see that https://search-test.library.ualberta.ca/ goes to `/symphony` Is that the best default?  I was thinking `/catalog` should be. What do you think?

I thought that symphony was a specialized version of catalog like journals, ebooks and databases but it's not.  Is there any value to keeping it? We can [continue to use](https://guides.rubyonrails.org/routing.html#specifying-a-controller-to-use) 'symphony' in the url if necessary.

File: Gemfile [hunk 2]
```diff
@@ -29,13 +29,8 @@ gem 'devise-guests'
 
 # Misc Utilities
 gem 'bootstrap_form', '~> 2.7.0'
-gem 'friendly_id'
 gem 'nokogiri', '~> 1.8.5'
 gem 'om'
-gem 'paperclip', '~> 6.1.0'
```
Comment (pgwillia) [inline Gemfile]: friendly_id and paperclip are only used by `profiles` as far as I can tell.

File: app/assets/stylesheets/ualib-custom.scss [hunk 3]
```diff
@@ -258,7 +258,7 @@ h2{
 /* Custom styles - Footer
 -------------------------------------------------- */
 .footer{
-    background: asset_url("footer-bg.jpg") #1b4724 repeat-x;
+    background: image_url("footer-bg.jpg") #1b4724 repeat-x;
```
Comment (pgwillia) [inline app/assets/stylesheets/ualib-custom.scss]: The images weren't showing up in development unless this.  Not sure about this change or why it wasn't needed elsewhere.

**Comment by pgwillia** [comment]
I'm not sure if I got the images :100: Didn't see the public images used anywhere.  Also many `app/assets/images`. I didn't go into the stylesheets, but these should be cleaned up too.

**Pull request merged**